src/createCollapsibleTabs.tsx

```diff
@@ -232,15 +235,18 @@ const createCollapsibleTabs = <T extends ParamList>() => {
               const nextValue = accDiffClamp.value + delta
               if (delta > 0) {
                 // scrolling down
-                accDiffClamp.value = Math.min(headerHeight || 0, nextValue)
```

Comment:
Did you tested the condition

```
if (accScrollY.value <= 0) {
  // handle overscroll on ios, when being dragged beyond the top border
  accDiffClamp.value = 0
```

Some lines before these changes?

Because I can't test on iOS to see how it is handling the overscroll.

Reply:
I wasn't able to validate it, also I'm not aware of what should be the behavior, i'm not an iOS user, but if you can explain the behavior I can validate/record it on a simulator

Reply:
😞  I can't test it. This is the behavior: https://medium.com/@wcandillon/ios-bounce-list-effect-with-react-native-5102e3a83999

Reply:
Just tested it, I believe the behavior is correct, but if someone else can also validate it would be great

Reply:
Let's merge, if there is an issue then, we fix it.